_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -120,9 +120,9 @@ release candidates.*
 
 - [BIPs #955][] updates [BIP174][] to standardize supplying hash preimages in
   [PSBT][topic psbt] input records. The standardization of these preimage fields
-  were [found to be necessary for miniscript aware finalizers][psbt ext], though
+  were [found to be necessary for miniscript-aware finalizers][psbt ext], though
```

Comment:
I still think s/were/was/ here, as the subject is the (singular) standardization of the fields, not the (plural) fields.

Reply:
Sorry, I looked right past that in your original feedback.